packages/react/cypress/support/auto.tsx

```diff
@@ -31,7 +31,8 @@ export const PolarisWrapper = ({ children }: { children: ReactNode }) => (
 );
 
 const suites: AutoSuiteConfig[] = [
-  { name: "MUI", adapter: MUIAdapter as any, wrapper: MUIWrapper },
+  // TODO: Uncomment this line when MUI autocompoents are ready for pre-release
+  // { name: "MUI", adapter: MUIAdapter as any, wrapper: MUIWrapper },
```

Comment:
Is it possible to keep the MUI tests around ?

Reply:
Great point: all cypress tests have been added back; they reference a `test-index.ts`.